### DISCUSSION TITLE
phpactor for php 7.2?

### DISCUSSION THREAD

**Discussion by Dante983** [phpactor/phpactor#2374] [Q&A] [answered]
Hello, as the title suggest, is it possible to use phpactor with php7.2 version?

I'ts not possible for me to downgrade the php version, but i would love to use phpactor.

Thanks.

**Comment by dantleech** [comment, marked as answer]
No, you need at least php 8.0 and preferably something more recent. I'd recommend updating your PHP version and/or using Docker for your projects or installing multiple versions of PHP :)

**Reply by Dante983** [reply]
Okay, got it, unfortunately I can't upgrade the php version, but thank you anyway. :D
